js/bootstrap-colorpicker-module.js

```diff
@@ -129,7 +129,7 @@ angular.module('colorpicker.module', [])
             if (Helper.stringParsers.hasOwnProperty(key)) {
               var parser = Helper.stringParsers[key];
               var match = parser.re.exec(val),
-                  values = match && parser.parse(match);
+              values = match && parser.parse(match);
```

Comment:
again. this is a multi-line variable definition and `values = match && parser.parse(match);` should be have 4 spaces if we are following best practices.